pallets/subtensor/src/coinbase/root.rs

```diff
@@ -299,24 +299,6 @@ impl<T: Config> Pallet<T> {
         Self::deposit_event(Event::NetworkRateLimitSet(limit));
     }
 
-    /// Checks if registrations are allowed for a given subnet.
-    ///
-    /// This function retrieves the subnet hyperparameters for the specified subnet and checks the `registration_allowed` flag.
-    /// If the subnet doesn't exist or doesn't have hyperparameters defined, it returns `false`.
```

Comment:
This is not dead code , and is required for not paying out emissions when registrations are turned off

Reply:
Where is it used? For all deleted code grep can't find any references inside the whole repo.

Reply:
in the coinbase. A couple of weeks ago , the commit for it got reverted , which is why you might have missed it in the dead code grep

Reply:
oh, ok

Reply:
returned it back

Reply:
still confused where can I find references to this method

Reply:
@gregzaitsev showed me the hotfix, I see now